config/drupal-phpunit-bootstrap-file.php

```diff
@@ -0,0 +1,168 @@
+<?php
+
+/**
+ * @file
+ *
+ * This fixes Drupal testing namespace autoloading and PHPUnit compatibility.
+ */
```

Comment:
🤔 I should probably specify this is mostly a copy job of `core/tests/bootstrap.php`